test-data/unit/check-varargs.test

```diff
@@ -448,6 +447,15 @@ def f(x: int, *, y: str) -> None: pass
 f(y='x', *(1,))
 [builtins fixtures/list.pyi]
 
+[case testVarArgsEmptyList]
```

Comment:
I assume that there's an existing test case that covers the empty tuple case?

Reply:
Actually, there is not. I thought that a tuple test is not in the scope of this PR.

Reply:
Since this changes the code related to handling empty/non-empty tuples as `*args`, it would be good to add a test case for this in case one doesn't exist already. This would validate that the change hasn't introduced a regression.

Reply:
Added.